WISE-Community/WISE
# Added a separate check to see if teacher notebook is enabled

How to test
1. Disable student notebook for a run
2. Enable teacher notebook for a run
3. Open CM for the run
4. Make sure that the teacher notebook is visible and you can take notes.
5. Make sure that the notebook icon in the left-hand side panel is hidden

Closes #1806
